app/pages/events/event_view.php

```diff
@@ -128,70 +125,64 @@
     <?php endif; ?>
 
     <?php if (count($event->races)): ?>
-        <h4>Courses : </h4>
-        <table role="grid">
-
-            <?php foreach ($event->races as $race):
-                $race_entry = $race->entries[0] ?? null; ?>
-                <details>
-                    <summary>
-                        <?= ConditionalIcon($race_entry && $race_entry->present) . " " . $race->name ?>
-                    </summary>
-                    <div class="row">
-                        <div class="col-sm-12 col-md-6">
-                            <ul class="fa-ul">
-                                <li><span class="fa-li"><i class="fa fa-calendar"></i></span>
-                                    <?= format_date($race->date) ?>
-                                </li>
-                                <li><span class="fa-li"><i class="fa fa-location-dot"></i></span>
-                                    <?= $race->place ?>
+        <h4>Courses :
+            <a href="#" title="Afficher tous les détails" onclick="
+                    htmx.findAll(htmx.closest(this, 'article'),'details').forEach(e => {e.open=!e.open});
+                    return false
+                ">
+                <i class="fa fa-up-down"></i>
```

Comment:
Great idea, love it ! I would rather use the `angles-down` icon from fa though, maybe like that :

<img width="756" alt="Capture d'écran 2023-12-16 195042" src="https://github.com/Atmos4/intranoseV2/assets/46241768/123e854e-8dff-4797-812e-fab162317791">

Reply:
Or like that : 

<img width="770" alt="Capture d'écran 2023-12-16 195416" src="https://github.com/Atmos4/intranoseV2/assets/46241768/58b23cc5-bb9f-47ba-8d54-f63c163d81d1">

Reply:
Okay for the current icon 🫡